lib/MusicBrainz/Server/Data/Artist.pm

```diff
@@ -222,6 +222,28 @@ sub find_by_work
     $self->query_to_list_limited($query, [($work_id) x 2], $limit, $offset);
 }
 
+sub find_by_ipi
```

Comment:
These should ideally go in `Data::Role::IPI` and `Data::Role::ISNI` respectively. And it might be better if they were plural & accepted multiple codes so that we don't have to execute SQL queries inside a loop.

Reply:
Hmm. Would you expect it to be just "is_ipi_in_use" or whatnot and only return a boolean (per code, or even in total) then? It would seem weird to have it return artists / labels from the role IMO but maybe not?

Reply:
Doesn't seem that weird to me. :) Why do you think so? We could do `is_ipi_in_use` etc. instead but I figured you were going to use the existing artists / labels for display. If not then that works.